PowerPointLabs/PowerPointLabs/DataSources/ColorDataSource.cs

```diff
@@ -259,7 +259,61 @@ public HSLColor ThemeColorTen
                 }
             }
         }
-        
+
+        private HSLColor themeColorElevenValue;
+
+        public HSLColor ThemeColorEleven
+        {
+            get
+            {
+                return themeColorElevenValue;
+            }
+            set
+            {
+                if (value != this.themeColorElevenValue)
+                {
+                    this.themeColorElevenValue = value;
+                    OnPropertyChanged("themeColorEleven");
```

Comment:
Seems like there are a lot of colors. I'm not sure if its possible (and not too difficult) to change this to use an array, but it would be good.

Reply:
This `ColorDataSource.cs` is mostly code from the old implementation of ColorsLab which i reused. Because all of them are using bindings with the UI and calling that `OnPropertyChanged` method, I don't think using a simple array will work for individual objects, i.e. they won't get updated as expected. 

One possible solution we can look into is using `ObservableCollections`, however I have not researched much into this just yet. Perhaps this could be something to consider refactoring in the near future, but as of now I don't think this affects functionality much!